style-dictionary/classic/colors.ts

```diff
@@ -82,7 +82,7 @@ const tokens: StyleDictionary.ColorsDictionary = {
   colorStrokeCodeEditorGutterActiveLineDefault: { dark: '{colorGrey650}' },
   colorTextAccent: { light: '{colorBlue600}', dark: '{colorBlue400}' },
   colorTextBreadcrumbCurrent: { dark: '{colorGrey300}' },
-  colorTextBreadcrumbIcon: { light: '{colorTextInteractiveDisabled}' },
+  colorTextBreadcrumbIcon: { light: '{colorGrey550}' },
```

Comment:
could we remove this token (`colorTextBreadcrumbIcon`) entirely, and always use `colorTextBreadcrumbCurrent`?

The values now match in classic light mode (as a result of this PR), are very similar in VR light, and in dark mode (classic & VR) the current icon colors don't seem to meet the 3:1 target either, so need changing one way or another

Reply:
Yes, my mistake - I was using the website playground as a reference, which has the wrong background colors.